test/components/structures/TimelinePanel-test.tsx

```diff
@@ -60,7 +63,7 @@ const newReceipt = (eventId: string, userId: string, readTs: number, fullyReadTs
 const getProps = (room: Room, events: MatrixEvent[]): TimelinePanel["props"] => {
     const timelineSet = { room: room as Room } as EventTimelineSet;
     const timeline = new EventTimeline(timelineSet);
-    events.forEach((event) => timeline.addEvent(event, true));
+    events.forEach((event) => timeline.addEvent(event, { toStartOfTimeline: true }));
```

Comment:
A side-issue which should be addressed elsewhere, but this will reduce the order of `events` in the resultant timeline, which feels pretty counter-intuitive.